Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -3785,6 +3785,26 @@ def batch(iterable, batch_size=1):
         current_batch = not_batched[:batch_size]
         not_batched = not_batched[batch_size:]
 
+def safe_get(dict_object, *keys, key_return_value = None):
+    """ Recursive safe get query, If keys found return value othewisw return None
+
+    Args:
+        key_return_value: Value to return when no key availble
+        dict_object: dictionary to query.
+        *keys: keys for recursive get.
+
+    Returns:
+        :return: Value from recursive get.
+        :rtype: ``obj``
+    """
+    for key in keys:
+        try:
+            dict_object = dict_object[key]
+        except KeyError:
```

Comment:
add TypeError, would cover all of our cases i think
![image](https://user-images.githubusercontent.com/37589583/84262797-f1ad6100-ab26-11ea-9592-8a58291f67ea.png)